resources/build/xcode-utils.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# This script will automatically have Xcode's build environment (and variables),
+# so there's no need to do anything extra to fetch them.
+
+# Takes one parameter:  "TAGGED".  Defaults to false (iOS: KMEI, SWKeyboard), sets tagged info if true (iOS: Keyman)
+#
+# Thanks to https://medium.com/@bestiosdevelope/automatic-build-incrementation-technique-for-ios-release-94eb0d08785b
+# for its documentation in this matter.
+function phaseSetBundleVersions() {
+  if [[ $# -gt 0 ]]; then
+    TAGGED=$1
+  else
+    TAGGED=false
+  fi
+
+  # For command-line builds, VERSION and VERSION_WITH_TAG) are forwarded through xcodebuild.
+  if [ -z $VERSION ]; then
```

Comment:
quotes needed

Reply:
Solving with https://stackoverflow.com/a/11369388.  I don't expect Xcode to use the `-u` unset-var check, but just in case, this will handle such cases.